source/interface/dav.js

```diff
@@ -21,6 +21,13 @@ function normaliseResponse(response) {
 
 function normaliseResult(result) {
     const { multistatus } = result;
+    if (multistatus === '') {
```

Comment:
Would this be better as a `!multistatus` check? This would catch occurences where `multistatus` might not be returned at all..

Reply:
well, it is bellow :)

Reply:
But bellow it throw, which is not good for empty multistatus. :thinking: